biz.aQute.bnd.maven/bnd.bnd

```diff
@@ -0,0 +1,38 @@
+# Set javac settings from JDT prefs
+-include: ${workspace}/cnf/includes/jdt.bnd
+
+Bundle-Name:            Shared logic used by bnd maven plugins
+Bundle-Description:	    Shared logic used by bnd maven plugins
+Bundle-DocURL:          http://www.aQute.biz/Code/Bnd
+-exportcontents: aQute.bnd.maven.lib.*
+
+Import-Package: \
+	org.apache.maven.*;provider=m2e,\
+	org.eclipse.aether.*;provider=m2e,\
+	*
+
+-buildpath: \
+	osgi.annotation;version=latest,\
+	biz.aQute.bndlib;version=latest,\
+	biz.aQute.resolve;version=latest,\
+	biz.aQute.repository;version=latest,\
+	org.apache.maven:maven-artifact,\
+	org.apache.maven:maven-core,\
+	org.apache.maven:maven-model,\
+	org.apache.maven:maven-plugin-api,\
+	org.apache.maven:maven-repository-metadata,\
+	org.apache.maven:maven-settings,\
+	org.eclipse.aether.api,\
+	org.slf4j.api;version=latest
+
+-conditionalpackage: \
+	aQute.lib*
+
+-testpath: \
+	${junit},\
+	${mockito},\
+	org.codehaus.plexus.classworlds,\
+	org.codehaus.plexus:plexus-utils,\
+	slf4j.simple;version=latest
+
+-fixupmessages.maven: "Export aQute.bnd.maven.lib.configuration...";is:=ignore
```

Comment:
Otherwise I get:
```
> Task :biz.aQute.bnd.maven:jar
warning: Export aQute.bnd.maven.lib.configuration,  has 1,  private references [aQute.lib.io]
```
which I believe is inert since `aQute.lib.io` is conditionally packaged into the bundle.